src/Transactions/TransactionDetails/TransactionInformation/TransactionInformation.js

```diff
@@ -13,19 +14,39 @@ import {
   FolioFormattedTime,
 } from '@folio/stripes-acq-components';
 
+import { getSourceLink } from './utils';
+
 const TransactionInformation = ({
   amount,
   currency,
   description,
   encumbrance,
   fiscalYearCode,
+  fiscalYearId,
   fromFundName,
+  invoiceId,
+  invoiceLineId,
   metadata,
   source,
   tags,
   toFundName,
   transactionType,
 }) => {
+  const sourceLink = useMemo(
```

Comment:
IMHO, source link looks like a component to me - it should return element or a simple text. No need to add that logic into TransactionInformation.js